ada/language/ast.py

```diff
@@ -11392,7 +11392,9 @@ def subp_decl_spec():
             Self.name_symbol, Self,
             dest_env=Entity.enum_type.primitives,
             metadata=T.Metadata.new(primitive=Entity.enum_type.node)
-        )
+        ),
+
+        add_env()
```

Comment:
Can you add a testcase for this? Also please add a comment, as it’s not clear why an env is needed here (EnumLiteralDecls can’t contain declarations, right?)